src/slowmokit/models/Kmeans.hpp

```diff
@@ -0,0 +1,80 @@
+#include "core.hpp"
+using namespace std;
```

Comment:
Do not use `using namespace std` anywhere, use `std::` wherever applicable instead